### ISSUE TITLE
Playstation Logo does not appear on start up in Wipeout XL

### ISSUE THREAD

**Issue by mra144910** [pathartl/BleemSync#176]
This is a followup to a previous issue I posted. I was having trouble getting Wipeout XL to start, but I managed to fix that by deleting the Japan version of the BIOS file. While the game loads and plays, the Playstation startup logo does not appear when starting up. It instead skips it entirely and starts at the games own intro screens(Things like the "Sony Computer Entertainment presents" screen or the opening intro).

While this may seem like a trivial problem, it does have an impact on using cheats in the game. For example I can't activate the cheat that turns ships into animals because that requires holding a button combination from the startup logo. I know this because I tested the same cheat on my PS3 and it only worked as stated above. I don't know if there is anyway to fix this.

**Comment by GMMan** [comment]
Deleting the BIOS means PCSX will use HLE. You need to have a valid BIOS file for the licensed screen to show up. Maybe try the worldwide one.

**Comment by mra144910** [comment]
Thanks for the reply. Just so you know I'm using a rom I ripped directly from the actual game disk using Imageburn. What exactly do you mean by the worldwide one? Is that a completely different version of the game that's separate from the one I own? Or is this something I can find within the games files? Pardon my newbie questions...

**Comment by GMMan** [comment]
The PSC comes with both a worldwide and a Japanese BIOS. If one isn't working for you, try the other.

**Comment by mra144910** [comment]
I just checked the BIOS folder. The only file in there is "romw.bin" which is what I assume you're talking about. I got rid of the Japanese file because that was preventing the game from launching. I don't know why this one isn't giving me the logo screen.

**Comment by GMMan** [comment]
Try renaming romw.bin to romJP.bin.

**Comment by GMMan** [comment]
You might be out of luck, then. Maybe try to find another BIOS.

**Comment by mra144910** [comment]
I think I found a fix. I found this reddit thread while looking up the game:

https://www.reddit.com/r/PlaystationClassic/comments/abeup4/is_wipeout_xl_wipeout_2097_working_for_anyone/

Near the bottom, someone mentions that replacing the romjp and romw with scph1001 will fix the problem. I got a hold of that file and that seemed to do the trick. However, it doesn't boot with scph1001 by default. I have to use the select+Triangle menu to set it to the correct BIOS and reboot the game. Doing so boots it the way it should, maybe a little too well as it also displays the white screen with "Sony Computer Entertainment" that normally shouldn't display. I'm not sure how to get it to boot with scph1001 by default, as it always goes back to romjp, even though I deleted that file.
